.circleci/config.yml

```diff
@@ -0,0 +1,232 @@
+version: 2.1
+
+aliases:
+  - &working_directory ~/merchant-center-application-kit
+
+  - &restore_yarn_cache
+    name: 'Restoring yarn cache'
+    keys:
+      - v1-yarn-cache-{{ checksum "yarn.lock" }}
+      - v1-yarn-cache
+
+  - &save_yarn_cache
+    name: 'Saving yarn cache'
+    key: v1-yarn-cache-{{ checksum "yarn.lock" }}
+    paths:
+      - ~/.cache/yarn
```

Comment:
```suggestion
      - ~/.cache/yarn
      - node_modules
```

Reply:
As mentioned on the other PR. Won't do. Can't find this as a recommendation on CircleCI's docs. It's a cache not the installed modules.